web/src/engine/keyboard-storage/src/cloud/queryEngine.ts

```diff
@@ -328,17 +328,17 @@ export default class CloudQueryEngine extends EventEmitter<EventMap> {
       comma=',';
     }
 
-    let errorStubs: ErrorStub[] = [];
     // Request keyboard metadata from the Keyman Cloud keyboard metadata server
     try {
-      let result: (KeyboardStub|ErrorStub)[]|Error = await this.keymanCloudRequest(cmd,false);
-      return mergeAndResolveStubPromises(result, errorStubs);
+      return await this.keymanCloudRequest(cmd,false);
     } catch(err) {
       // We don't have keyboard info for this ErrorStub
-      console.error(err);
-      let stub: ErrorStub = {error: err};
-      errorStubs.push(stub);
-      return Promise.reject(errorStubs);
+      if(err instanceof Error) {
+        console.error(`Error for request command-string ${cmd}: ${err.message}\n  ${err.stack}`);
+      } else {
+        console.error(err);
+      }
```

Comment:
The `console.error` statement here is directly responsible for the second example report above and for the breadcrumb on the first example.